frontend/src/components/Pages/MissionPage/TaskOverview/TaskTable.tsx

```diff
@@ -5,44 +5,27 @@ import { useLanguageContext } from 'components/Contexts/LanguageContext'
 import { Task, TaskStatus } from 'models/Task'
 import { tokens } from '@equinor/eds-tokens'
 import { getColorsFromTaskStatus } from 'utils/MarkerStyles'
-import { StyledTableBody, StyledTableCaptionGray, StyledTableCell } from 'components/Styles/StyledComponents'
 
 const StyledTable = styled(Table)`
     display: block;
     overflow: auto;
     max-width: calc(80vw);
 `
-const StyledTypography = styled(Typography)`
-    font-family: Equinor;
-    font-size: 28px;
-    font-style: normal;
-    line-height: 35px;
-
-    @media (max-width: 500px) {
-        font-size: 24px;
-        line-height: 30px;
-    }
-
-    padding-bottom: 10px;
-`
 
 export const TaskTable = ({ tasks }: { tasks: Task[] | undefined }) => {
     const { TranslateText } = useLanguageContext()
     return (
         <StyledTable>
-            <StyledTableCaptionGray>
-                <StyledTypography variant="h2">{TranslateText('Tasks')}</StyledTypography>
-            </StyledTableCaptionGray>
             <Table.Head>
                 <Table.Row>
-                    <StyledTableCell>#</StyledTableCell>
-                    <StyledTableCell>{TranslateText('Tag-ID')}</StyledTableCell>
-                    <StyledTableCell>{TranslateText('Description')}</StyledTableCell>
-                    <StyledTableCell>{TranslateText('Inspection Types')}</StyledTableCell>
-                    <StyledTableCell>{TranslateText('Status')}</StyledTableCell>
+                    <Table.Cell>#</Table.Cell>
```

Comment:
Consider using different design for task table on small screen
<img width="330" alt="image" src="https://github.com/user-attachments/assets/ed8102af-1227-45de-8135-b082952bfde0">